cli/src/pcluster/models/cluster.py

```diff
@@ -1081,3 +1083,21 @@ def get_log_events(
     def _stack_events_stream_name(self):
         """Return the name of the stack events log stream."""
         return STACK_EVENTS_LOG_STREAM_NAME_FORMAT.format(self.stack_name)
+
+    def _validate_scheduling_update(self, changes, target_config):
+        """Update of Scheduling is not supported when SupportsClusterUpdate of the scheduler plugin is set to false."""
+        if (
+            self.config.scheduling.scheduler == "plugin"
+            and self.config.scheduling.settings.scheduler_definition.requirements
+            and self.config.scheduling.settings.scheduler_definition.requirements.supports_cluster_update is False
+        ) and target_config.source_config.get("Scheduling") != self.config.source_config.get("Scheduling"):
+            scheduling_changes = []
+            for change in changes[1:]:
+                if change[0][0] == "Scheduling":
+                    scheduling_changes.append(change)
+            if len(scheduling_changes) >= 1:
```

Comment:
Yes, it is possible, for example: {queues: [q1, q2]} is not equal to {queues: [q2, q1]},  but it doesn’t matter here because we also check changes in scheduling to check if there’s scheduling update. Add a comment in the code to explain that